ui/widgets/autocomplete.js

```diff
@@ -267,8 +268,11 @@ $.widget( "ui.autocomplete", {
 				// Announce the value in the liveRegion
 				label = ui.item.attr( "aria-label" ) || item.value;
 				if ( label && String.prototype.trim.call( label ).length ) {
-					this.liveRegion.children().hide();
-					$( "<div>" ).text( label ).appendTo( this.liveRegion );
+					clearTimeout( this.liveRegionTimer );
+					var that = this;
+					this.liveRegionTimer = setTimeout( function() {
```

Comment:
Please try using `this._delay` here, see https://github.com/jquery/jquery-ui/blob/e90096e9dd25392118c2c578f490445870ced686/ui/widgets/autocomplete.js#L410-L411 for an example!

Reply:
Looks like it works well, thanks for pointing that out!